### PR TITLE
Reduce max memory for Solr on Vagrant

### PR DESCRIPTION
**Pull request by hackdna**

### CODE REVIEW

File: deployment/refinery-modules/refinery/manifests/solr.pp [hunk 1]
```diff
@@ -39,10 +39,19 @@
...
-    file_line { "solr_config_home":
-      path  => "/var/solr/solr.in.sh",
-      line  => "SOLR_HOME=${django_root}/solr",
-      match => "^SOLR_HOME",
+    file_line {
+      "solr_config_home":
+        path  => "/var/solr/solr.in.sh",
+        line  => "SOLR_HOME=${django_root}/solr",
+        match => "^SOLR_HOME";
+      'solr_config_heap':
```
Comment (scottx611x) [inline deployment/refinery-modules/refinery/manifests/solr.pp]: Double quotes for consistency?

File: deployment/refinery-modules/refinery/manifests/solr.pp [hunk 2]
```diff
@@ -39,10 +39,19 @@
...
+        path    => '/var/solr/solr.in.sh',
+        line    => 'SOLR_HEAP="32m"',
+        match   => 'SOLR_HEAP=',
+        replace => $::deployment_platform ? {
```
Comment (scottx611x) [inline deployment/refinery-modules/refinery/manifests/solr.pp]: This is cool, had no clue we could do that

File: deployment/refinery-modules/refinery/manifests/solr.pp (hunk 1 above)
Comment (hackdna) [reply deployment/refinery-modules/refinery/manifests/solr.pp]: Actually, they should all be single quotes unless variable interpolation required :) https://docs.puppet.com/puppet/5.1/style_guide.html#quoting

Comment (scottx611x) [reply deployment/refinery-modules/refinery/manifests/solr.pp]: Okay cool. Could you update the other double quotes w/o interpolated vars?

Comment (hackdna) [reply deployment/refinery-modules/refinery/manifests/solr.pp]: In all the manifests? :)

Comment (scottx611x) [reply deployment/refinery-modules/refinery/manifests/solr.pp]: If you're feeling up to it I guess? But I was just thinking about ones from this diff.

**Pull request merged**